src/lib/Preview.js

```diff
@@ -987,6 +986,12 @@ class Preview extends EventEmitter {
         // Add the response interceptor to the preview instance
         this.options.responseInterceptor = options.responseInterceptor;
 
+        this.options.annotationsOptions = merge(options.annotationsOptions, {
```

Comment:
Should we combine `this.options. showAnnotationsControls` into the new `this.options.annotationsOptions`?

Reply:
I think `showAnnotationsControls` is preview-specific and shouldn't be passed on to the annotations instance. It would also increase the risk of the change.

Reply:
Yeah, I figured that seemed specific to the preview controls and not really an option for the annotations sdk